app/modules/finance/subcategories/router.py

```
@@ -19,3 +21,22 @@ async def create_subcategory(
+    service: Annotated[SubcategoryService, Depends(get_subcategory_service)],
+    user: Annotated[User, Depends(get_current_user)],
+):
+    return await service.get_by_id(subcategory_id)
+
+
+@router.get("/list/{category_id}", response_model=Page[SubcategoryBase])
+async def list_by_category(
+    category_id: uuid.UUID,
+    params: Annotated[Params, Depends()],
+    service: Annotated[SubcategoryService, Depends(get_subcategory_service)],
+    user: Annotated[User, Depends(get_current_user)],
```

Ruff (0.15.2)

[warning] Unused function argument: `user`

(ARG001)